[BUG] Server Crashing With Turkish Locale

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Server crashing before mod loading when run with Turkish Locale.

### Expected Behavior

Server running without issues.

### Steps To Reproduce

1. Create Forge server with Ender-IO 1.20.1 6.0.5
2. Run the server with default settings and Turkish Locale

### Environment

```markdown
- OS: Windows 11
- Java version: OpenJDK 17.0.7
- Minecraft version: 1.20.1
- Version: 6.0.5
```


### Anything else?

[latest.log](https://github.com/SleepyTrousers/EnderIO-Rewrite/files/12069927/latest.log)
[debug.log](https://github.com/SleepyTrousers/EnderIO-Rewrite/files/12069928/debug.log)

`Caused by: net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: enderio:capacitor.energy_capac\u0131ty
`
"\u0131" being the Turkish character "ı".

--- a/src/api/java/com/enderio/api/capacitor/CapacitorModifier.java
+++ b/src/api/java/com/enderio/api/capacitor/CapacitorModifier.java
@@ -2,6 +2,8 @@ package com.enderio.api.capacitor;
 
 import net.minecraft.resources.ResourceLocation;
 
+import java.util.Locale;
+
 /**
  * Capacitor key types, for use in loot capacitors for targeting increases to general stats.
  */
@@ -16,6 +18,6 @@ public enum CapacitorModifier {
     public final ResourceLocation id;
 
     CapacitorModifier() {
-        this.id = new ResourceLocation("enderio", "capacitor." + name().toLowerCase());
+        this.id = new ResourceLocation("enderio", "capacitor." + name().toLowerCase(Locale.ROOT));
     }
 }

--- a/src/machines/java/com/enderio/machines/data/recipes/SagMillRecipeProvider.java
+++ b/src/machines/java/com/enderio/machines/data/recipes/SagMillRecipeProvider.java
@@ -25,6 +25,7 @@ import net.minecraftforge.registries.ForgeRegistries;
 
 import java.util.HashSet;
 import java.util.List;
+import java.util.Locale;
 import java.util.Set;
 import java.util.function.Consumer;
 
@@ -414,7 +415,7 @@ public class SagMillRecipeProvider extends EnderRecipeProvider {
             json.add("input", input.toJson());
             json.addProperty("energy", energy);
             if (bonusType != BonusType.MULTIPLY_OUTPUT) {
-                json.addProperty("bonus", bonusType.toString().toLowerCase());
+                json.addProperty("bonus", bonusType.toString().toLowerCase(Locale.ROOT));
             }
 
             JsonArray outputJson = new JsonArray();

